docs/helm.md

```diff
@@ -0,0 +1,17 @@
+# Helm Support in Tilt
+
+Tilt has native support for Helm! To run `helm template` on a chart directory from Tilt simply pass the chart directory to the `helm` built-in like so:
+
+```python
+yml = helm('path/to/chart')
+# The result is just YAML, which you can use throughout Tilt anywhere you would normally use YAML
+k8s_yaml(yml)
```

Comment:
maybe add a section on how to add a Dockerfile to substitute in your own build process?

this could largely be copy-and-pasted from another part of the guide